static/app/components/events/interfaces/spans/index.tsx

```diff
@@ -76,11 +76,15 @@ class SpansInterface extends PureComponent<Props, State> {
       return null;
     }
 
-    const label = tn(
-      'There is an error event associated with this transaction event.',
-      `There are %s error events associated with this transaction event.`,
-      errors.length
-    );
+    // This is intentional as unbalanced string formatters in `tn()` are problematic
+    const label =
+      errors.length === 1
+        ? t('There is an error event associated with this transaction event.')
+        : tn(
+            `There are %s error events associated with this transaction event.`,
+            `There are %s error events associated with this transaction event.`,
+            errors.length
+          );
```

Comment:
```suggestion
        : t(`There are %s error events associated with this transaction event.`, errors.length);
```

🤷‍♀️

Reply:
It looks like certain languages have different pluralizations (e.g. Russian has one for when the number ends in `[1]`, `[2, 3, 4]`, and `[5, 6, 7, 8, 9, 0]`)

Reply:
The current diff is correct, you need multiple messages to work with the various pluralization rules.

Reply:
that's interesting...thanks

Reply:
tried it locally and it looks good